net.sourceforge.pmd.eclipse.plugin/src/main/java/net/sourceforge/pmd/eclipse/ui/preferences/RuleTableViewerSorter.java

```diff
@@ -148,9 +148,9 @@ public void setSortDescending(boolean sortDescending) {
     /**
      * @return Returns the comparator.
      */
-    public Comparator<Rule> getComparator() {
-        return comparator;
-    }
+//    public Comparator<Rule> getComparator() {
```

Comment:
Please keep this in. I know, in eclipse this is highlighted as an error, but I think, the problem is due to incorrect eclipse API configuration. I'm planning to increase the minimum required eclipse version, this plugins runs with, and hopefully this issue will be fixed then.